persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -232,4 +252,27 @@ public void printLongDesc(StringBuilder out) {
     public void printUsage(StringBuilder stringBuilder) {
         stringBuilder.append("  ballerina " + COMPONENT_IDENTIFIER + " db push").append(System.lineSeparator());
     }
+
+    private Path getDriverPath() throws BalException {
+        String relativeLibPath;
+
+        DependencyGraph<ResolvedPackageDependency> resolvedPackageDependencyDependencyGraph =
+                balProject.currentPackage().getResolution().dependencyGraph();
+
+        ResolvedPackageDependency root = resolvedPackageDependencyDependencyGraph.getRoot();
+
+        Package mysql = resolvedPackageDependencyDependencyGraph.getDirectDependencies(root).stream().
+                filter(resolvedPackageDependency -> resolvedPackageDependency.packageInstance().
+                descriptor().toString().contains(BALLERINA_MYSQL_DRIVER_NAME)).findFirst().get().packageInstance();
+
+        List<Map<String, Object>> dependencies = mysql.manifest().platform(PLATFORM).dependencies();
+
+        for (Map<String, Object> dependency : dependencies) {
+            if (dependency.get(PATH).toString().contains(MYSQL_CONNECTOR_NAME_PREFIX)) {
+                relativeLibPath = dependency.get(PATH).toString();
+                return mysql.project().sourceRoot().resolve(relativeLibPath);
+            }
+        }
```

Comment:
Shall we add all dependencies to the classpath? In MySQL, we have only one jar to add, but in other DBS, there may have multiple jars

Reply:
In the current implementation, the parent dir of the jar[[1]](https://github.com/ballerina-platform/persist-tools/blob/bf7200da5d01ea43bc5c86c115b324297a06bc57/persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java#L195) is sent to the function where the classpath is updated. In that function, all the jars are added[[2]](https://github.com/ballerina-platform/persist-tools/blob/e37f984f0c0d2ebfe6b1478600acc2f4ac3f9dab/persist-cli/src/main/java/io/ballerina/persist/utils/JdbcDriverLoader.java#L44). 

[1] https://github.com/ballerina-platform/persist-tools/blob/bf7200da5d01ea43bc5c86c115b324297a06bc57/persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java#L195
[2] https://github.com/ballerina-platform/persist-tools/blob/e37f984f0c0d2ebfe6b1478600acc2f4ac3f9dab/persist-cli/src/main/java/io/ballerina/persist/utils/JdbcDriverLoader.java#L44